uploader.json

```diff
@@ -45,6 +45,11 @@
             "pattern": [],
             "symbolic_name": "ceph_v"
         },
+        {
+            "command": "/usr/bin/ceph insights",
```

Comment:
@noahdesu your command should be `"command": "python -m insights.tools.cat ceph_insights"`.  This will run your `ceph_insights` command in insights/specs/default.py, using the datasource dependency that you included.